playbooks/robusta_playbooks/pod_troubleshooting.py

```diff
@@ -1,9 +1,30 @@
 # TODO: move the python playbooks into their own subpackage and put each playbook in its own file
+import logging
+
 import humanize
 from robusta.api import *
 from typing import List
 
 
+class StackTraceObject:
```

Comment:
Can we use Pydantic here? 

class StackTraceObject(BaseModel):
    time: float = None
    status: str = None
    error: str = None
    trace: str = None

Any reason not to?